projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -34,6 +34,7 @@
 			),
 			'authors_only'    => '(bool) Set to true to fetch authors only',
 			'include_viewers' => '(bool) Set to true to include viewers for Simple sites. When you pass in this parameter, order, order_by and search_columns are ignored. Currently, `search` is limited to the first page of results.',
+			'total_users'     => '(bool) Set to true to return the total number of site users.',
```

Comment:
This flag makes the total users count opt-in. 

This is a micro optimization as `count_users` can potentially be CPU intensive, albeit in rare cases: [core.trac.wordpress.org/ticket/29785#comment:19](https://core.trac.wordpress.org/ticket/29785#comment:19)